### PR TITLE
Add backwards cases for Moonriver xcm

### PR DESCRIPTION
**Pull request by leohar**
- add BNC from Moonriver to Karura
- add MOVR from Bifrost to Moonriver

### CODE REVIEW

File: xcm/transfers_dev.json [hunk 1]
```diff
@@ -480,6 +508,34 @@
...
       ]
+    },
+    {
+      "chainId": "9f28c6a68e0fc9646eff64935684f6eeeece527e37bbe1f213d22caa1d9d6bed",
+      "assets": [
+        {
+          "assetId": 8,
+          "assetLocation": "MOVR",
+          "assetLocationPath": {
+            "type": "absolut"
```
Comment (valentunn) [inline xcm/transfers_dev.json:519]: Typo

File: xcm/transfers_dev.json [hunk 2]
```diff
@@ -183,6 +183,34 @@
               "type": "xtokens"
             }
           ]
+        },
+        {
+          "assetId": 5,
+          "assetLocation": "BNC",
+          "assetLocationPath": {
```
[the remark this answers is not in the document]
Comment (leohar) [reply xcm/transfers_dev.json:190]: done

File: xcm/transfers_dev.json [hunk 3]
```diff
@@ -480,6 +503,34 @@
...
+          "assetId": 8,
+          "assetLocation": "MOVR",
+          "assetLocationPath": {
+            "type": "absolute"
+          },
+          "xcmTransfers": [
+            {
+              "destination": {
+                "chainId": "401a1f9dca3da46f5c4091016c8a2f26dcea05865116b286f60f668207d1474b",
+                "assetId": 5,
```
Comment (ERussel) [inline xcm/transfers_dev.json:520]: I think MOVR asset id on Moonriver is 0

Comment (leohar) [reply xcm/transfers_dev.json]: you're absolutely right

File: xcm/transfers_dev.json [hunk 4]
```diff
@@ -480,6 +503,34 @@
...
+            "type": "absolute"
+          },
+          "xcmTransfers": [
+            {
+              "destination": {
+                "chainId": "401a1f9dca3da46f5c4091016c8a2f26dcea05865116b286f60f668207d1474b",
+                "assetId": 0,
+                "fee": {
+                  "mode": {
+                    "type": "proportional",
```
Comment (ERussel) [inline xcm/transfers_dev.json:523]: Let's put standard here. I think we can make a rule: if the receiving token is native for destination then fee.mode.type is always `standard`

File: xcm/transfers_dev.json [hunk 5]
```diff
@@ -183,6 +183,29 @@
...
+          },
+          "xcmTransfers": [
+            {
+              "destination": {
+                "chainId": "baf5aabe40646d11f0ee8abbdc64f4a4b7674925cba08e4a05ff9ebed6e2126b",
+                "assetId": 4,
+                "fee": {
+                  "mode": {
+                    "type": "proportional",
+                    "value": "1"
```
Comment (ERussel) [inline xcm/transfers_dev.json:201]: Let's put `1854040649841248`

```
KAR precision 12

pub fn base_tx_in_kar() -> Balance {
	cent(KAR) / 10 // 0.1 KAR
}

pub const WEIGHT_PER_SECOND: Weight = 1_000_000_000_000;
pub const WEIGHT_PER_MILLIS: Weight = WEIGHT_PER_SECOND / 1000; // 1_000_000_000
pub const WEIGHT_PER_MICROS: Weight = WEIGHT_PER_MILLIS / 1000; // 1_000_000
pub const WEIGHT_PER_NANOS: Weight = WEIGHT_PER_MICROS / 1000; // 1_000
pub const ExtrinsicBaseWeight: Weight = 86_298 * WEIGHT_PER_NANOS;

pub fn kar_per_second() -> u128 {
	// WEIGHT_PER_NANOS = WEIGHT_PER_MICROS / 1000 = WEIGHT_PER_MILLIS / 1_000_000 = WEIGHT_PER_SECOND / 1_000_000_000
	// base_weight = 86_298 * WEIGHT_PER_SECOND / 1_000_000_000
	let base_weight = Balance::from(ExtrinsicBaseWeight::get());

	// base_tx_per_second = WEIGHT_PER_SECOND / (86_298 * WEIGHT_PER_SECOND / 1_000_000_000) = 1_000_000_000 / 86_298
	let base_tx_per_second = (WEIGHT_PER_SECOND as u128) / base_weight;

	// 100_000_000_000 * 1_000_000_000 / 86_298
	base_tx_per_second * base_tx_in_kar()
}

pub fn ksm_per_second() -> u128 {
	// 100_000_000_000 * 1_000_000_000 / (86_298 * 50) ~ 23175508123016
	kar_per_second() / 50
}

// 100_000_000_000 * 1_000_000_000 * 80 / (86_298 * 50) = 100_000_000_000_000_000_000 * 8 / (86_298 * 5) ~ 1854040649841248
bnc_per_second = ksm_per_second() * 80
```

**Review by ERussel — APPROVED** [review]

**Pull request merged**